lib/apiservers/engine/backends/network.go

```diff
@@ -178,6 +178,18 @@ func (n *Network) ConnectContainerToNetwork(containerName, networkName string, e
 		nc.Address = &endpointConfig.IPAMConfig.IPv4Address
 	}
 
+	log.Debugf("EndpointsConfig: %#v", endpointConfig)
```

Comment:
looks identical to the backends/container.go fragment - do these need to be consistent if changes are made? If so this should be a util function